python_ta/transforms/type_inference_visitor.py

```diff
@@ -663,9 +667,11 @@ def visit_classdef(self, node: astroid.ClassDef) -> None:
     def get_attribute_class(self, t: type) -> Tuple[str, type, bool]:
         is_inst_expr = True
 
+        # Class type: e.g., 'Type[_ForwardRef('A')]'
         if hasattr(t, '__name__') and t.__name__ == 'Type':
             class_type = t.__args__[0]
             is_inst_expr = False
+        # Class instance or builtin type; e.g., '_ForwardRef('A)' or 'int'
```

Comment:
There's a missing apostrophe in this comment. Also the comment is a bit misleading: really, this case is for "instance of class or builtin type"